docs/using_tags.md

```diff
@@ -0,0 +1,87 @@
+# Using Tags
+
+For understanding and taking the most advantage of `ezplatform-http-cache`, understanding tags is key _(pun intended)_.
```

Comment:
`Understanding tags is the key _(pun intended)_ to making the most of...`

Reply:
skipping the joke, not everyone will get it and I don't want to waste time explaining tags is a key also inside the joke as the joke is then moot :)